src/patterns/ffi/wrappers.md

```diff
@@ -153,7 +153,7 @@ optimizations that C code cannot attain. Being denied access to certain
 shortcuts is the price Rust programmers need to pay.
 
 [^1]: For the C programmers out there scratching their heads, the iterator need
-not be read *during* this code cause the UB. The exclusivity rule also enables
+not be read *during* this code to cause the UB. The exclusivity rule also enables
```

Comment:
I hope I understood that correctly. It might be better to expand this text and moving it from the footnote.